### PR TITLE
FAI-611: Trusty UI: Toolbars weird extra spacing after PF update

### PR DESCRIPTION
**Pull request by kelvah**
See: https://issues.redhat.com/browse/FAI-611

The main reason behind this PR was increasing the vertical space available to the Counterfactual Table. Given the extra padding problem is caused by PF I decided to fix it globally instead of only addressing it inside CF. It's affecting also the Toolbar's in the audit overiew.

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Pull Request</b>  
  Please add comment: <b>Jenkins retest this</b>
 
* <b>Quarkus LTS checks</b>  
  Please add comment: <b>Jenkins run LTS</b>

* <b>Native checks</b>  
  Please add comment: <b>Jenkins run native</b>

</details>

### CODE REVIEW

File: ui-packages/packages/trusty/src/components/Templates/TrustyApp/TrustyApp.scss [hunk 1]
```diff
@@ -1,3 +1,12 @@
 .trusty-logo {
   width: 158px;
 }
+
+/*
+  Fixing weird toolbar extra spacing introduced after updating to PF 2021.08 (see FAI-611)
+  The issue is not present in more recent versions of PF.
+  To be removed after the next PF update on kogito-apps.
```
Comment (manstis) [inline ui-packages/packages/trusty/src/components/Templates/TrustyApp/TrustyApp.scss:8]: How/where do you track this?

**Review by manstis — APPROVED** [review]
The change looks (and works) good for me.

I just made a comment about administering the future rollback.

File: ui-packages/packages/trusty/src/components/Templates/TrustyApp/TrustyApp.scss (hunk 1 above)
Comment (kelvah) [reply ui-packages/packages/trusty/src/components/Templates/TrustyApp/TrustyApp.scss:8]: I've created [a new ticket](https://issues.redhat.com/browse/FAI-651) to follow up on this.

**Comment by manstis** [comment]
Oh @kelvah you really are naughty.

Every one of your recent PRs lacks sufficient unit test coverage. You really must try harder ;-)

@mbiarnes Hello.. @kelvah seems to be suffering a continual problem where SonarCloud is moaning about insufficient test  coverage; however the files it complains about are not changed by his PRs and hence should not be included. Is this a known issue!?!?

**Review by sjamboro — APPROVED** [review]

**Pull request merged**